cli/ts/commands/verify.ts

```diff
@@ -117,6 +128,47 @@ export const verify = async (
   if (onChainTallycomment !== newTallyCommitment) logError("The on-chain tally commitment does not match.");
   logGreen(quiet, success("The on-chain tally commitment matches."));
 
+  // verify total spent voice credits on-chain
+  const isValid = await tallyContract.verifySpentVoiceCredits(
+    tallyResults.totalSpentVoiceCredits.spent,
+    tallyResults.totalSpentVoiceCredits.salt,
+    newResultsCommitment,
+    newPerVOSpentVoiceCreditsCommitment,
+  );
+  if (isValid) {
+    logGreen(quiet, success("The on-chain verification of total spent voice credits passed."));
+  } else {
+    logError("The on-chain verification of total spent voice credits failed.");
+  }
+
+  // verify per vote option voice credits on-chain
+  const failedSpentCredits = await verifyPerVOSpentVoiceCredits(
+    tallyContract,
+    tallyResults,
+    voteOptionTreeDepth,
+    newSpentVoiceCreditsCommitment,
+    newResultsCommitment,
+  );
+  if (failedSpentCredits.length === 0) {
+    logGreen(quiet, success("The on-chain verification of per vote option spent voice credits passed"));
+  } else {
+    logError(`These spent voice credits failed on-chain verifications ${failedSpentCredits}`);
```

Comment:
I think "At least one spent voice credits entry in the tally results failed the on-chain verification. Please check your tally results at these indexes: ${failedSpentCredits}" would sound a bit better and give more troubleshooting info to users.